### ISSUE TITLE
The variable's value is assigned but never used

### ISSUE THREAD

**Issue by AppChecker** [NagiosEnterprises/ndoutils#39]
Hi!

Please look [this code fragment](https://github.com/NagiosEnterprises/ndoutils/blob/c115d3dea3b1040c67d287671486a82a6af55ced/src/ndomod.c#L3396-L3397)
```
			last_state=temp_host->last_state;
			last_state=temp_host->last_hard_state;
```
Varialbe 'last_state' is assigned twice.
Probably, it should be:
```
			last_state=temp_host->last_state;
			last_hard_state=temp_host->last_hard_state;
```
like [this](https://github.com/NagiosEnterprises/ndoutils/blob/c115d3dea3b1040c67d287671486a82a6af55ced/src/ndomod.c#L3404-L3405)

This possible defect found by [AppChecker](https://npo-echelon.ru/en/solutions/appchecker.php)

**Comment by hedenface** [comment]
@Madlohe or @corynorell Can one of you create a patch and file a PR please?

**Comment by hedenface** [comment]
Closed via #40 - @Madlohe can you add yourself to the contributers/thanks list? Thanks.
